### ISSUE TITLE
grainy textures

### ISSUE THREAD

**Issue by kringlexd** [JarateKing/CleanTF2plus#17]
the textures look grainy

**Comment by Tkelloww** [comment]
Read TROUBLESHOOTING.md to understand why. 

OR

Read this (a cnp of it)
## Grainey textures

> When I enable flat textures, instead of being one color they are lots of lines and grainey textures

This is the result of resized flat textures on sv_pure -- it's the original image, except resized down from 512x512 to 1x1 (if you look closely, the 'line' effect is actually just very small textures). The fix for this is to not use the resized option, so that the original size is preserved with the flat textures. This is a tradeoff between returning to default textures in sv_pure 1 or 2, but having a much larger filesize.

**Comment by Tkelloww** [comment]
> Read TROUBLESHOOTING.md to understand why.
> 
> OR
> 
> Read this (a cnp of it)
> 
> ## Grainey textures
> > When I enable flat textures, instead of being one color they are lots of lines and grainey textures
> 
> This is the result of resized flat textures on sv_pure -- it's the original image, except resized down from 512x512 to 1x1 (if you look closely, the 'line' effect is actually just very small textures). The fix for this is to not use the resized option, so that the original size is preserved with the flat textures. This is a tradeoff between returning to default textures in sv_pure 1 or 2, but having a much larger filesize.

Yes im sorry i know this is late (a year gap) i just downloaded this and had a similar problem and just looked in the files
